src/test/extensions/manifest.spec.ts

```diff
@@ -34,6 +36,24 @@ describe("manifest", () => {
     });
   });
 
+  describe(`${manifest.getInstanceRef}`, () => {
```

Comment:
Won't this print out the function object? I think this probably should be describe("getInstanceRef"... instead

Reply:
oh added `.name` to the end now it works. I like this pattern because it gives better IDE support. (renaming, clicking through, etc..) If you think we should follow the rest of the codebase I'm happy to change it too